graph/src/blockchain/mod.rs

```diff
@@ -51,6 +51,12 @@ use self::{
     client::ChainClient,
 };
 
+#[async_trait]
+pub trait BlockIngestor: 'static + Send + Sync {
+    async fn run(self: Box<Self>);
+    fn network_name(&self) -> String;
```

Comment:
This change will create issues with the reference since it's not 'static and it's an async context, will end up forcing to_string() being called elsewhere, prolly not worth the optimisation.

Reply:
Got it! Resolving.